### DISCUSSION TITLE
Searching For Exploits; (version 119)

### DISCUSSION THREAD

**Discussion by Sxri** [3kh0/ext-remover#863] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi. I'm wondering if there are any alternative exploits for Chrome OS, specifically version 119. Skiovox has been patched via Chrome in this version, and the majority of the "chrome://" tabs are blocked as well by administrators. If anyone has any suggestions on possible methods to begin an exploit on this version so I can create it, or if you already have a premade exploit that is available I would greatly appreciate it. Thank you.

(I've tried all possible methods listed in this repository so far in the ReadMe file, none of which have worked)

**Comment by akabutnicer** [comment]
Your case is pretty iffy, but regardless, there's a few options which may or may not help. You could try to replace the `/Extensions` directory via VMC crosh exploit (which I cannot give details on since I am not in TN). You could also try Dextensify (which isn't recommended)

**Comment by akabutnicer** [comment]
And Skiovox, as you said, "via Chrome", is wrong. The version which Chrome will patch Skiovox is **121**. Therefore, whatever supposedly "patched" Skiovox, must have been on your school's side, which even then they probably didn't even thoroughly patch the exploit. I recommend you keep trying because there isn't much an admin can do as far as patching Skiovox.

**Reply by Sxri** [reply]
Basically the "Patch" I figured was because everything worked, up until the part where you click WiFi > Open in settings. When trying to click the "Open in settings HREF link, it does nothing. It does the effect of clicking where it highlights, but it does not do anything beyond that. Not sure if there is a way around this, but this was the same for the Geogebra method, when trying to click links it just refrains from redirecting or completing the action. So either its a patch on the creators end (in this case Chrome OS), which you said wasn't. Or my school somehow did this. If so, any way around this would be great. I'm just not sure how.

**Reply by PenguinLord561** [reply]
@Sxri Yeah, this happens to me too. I tried a lot of stuff but none of them really works.

**Comment by Su4o** [comment]
If your that intended to do an exploit, you can wait for cryptosmite (the new exploit that will come in the year 8098, _very advanced or something like that_) 












_Or bring your ipad..._

**Reply by akabutnicer** [reply, marked as answer]
Since when was intented a word bro :sob:

**Reply by Su4o** [reply]
@akabutnicer I never seen that until u told me. XD

i changed it to intended now
